ui/components/app/assets/nfts/nft-details/nft-full-image.tsx

```diff
@@ -94,7 +107,7 @@ export default function NftFullImage() {
               size={ButtonIconSize.Sm}
               ariaLabel={t('back')}
               iconName={IconName.Close}
-              onClick={() => history.push(`${ASSET_ROUTE}/${asset}/${id}`)}
```

Comment:
Key fix: we were missing the chainId on the path, so now that the details page uses it (and attempt to convert to CAIP/HEX) it crashes if it is missing.